common/src/main/java/com/wynntils/core/features/FeatureRegistry.java

```diff
@@ -94,7 +94,7 @@ public static void init() {
         registerFeature(new HealthPotionBlockerFeature());
         registerFeature(new HidePotionGlintFeature());
         registerFeature(new InfoMessageFilterFeature());
-        registerFeature(new IngredientPouchHotkeyFeature());
```

Comment:
Did you intend to remove this feature?

Reply:
No, wtf did I do lol

Reply:
Probably double clicked the feature name and replaced it with your feature name..?